### PR TITLE
anaconda buildpack for Python3

### PR DESCRIPTION
**Pull request by beholders-eye**
My first buildpack, I tested it with https://github.com/project-kotinos/lneuhaus___pyrpl/, and it worked.

### CODE REVIEW

**Comment by johnewart** [comment]
Hey this will work for one Adam was doing — I was just going to write one today so that’s awesome! 🤘🏻

File: buildpacks/anaconda.go [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+)
+
+type AnacondaBuildTool struct {
+	BuildTool
+	version string
+	spec    BuildToolSpec
+}
+
+// Fixed for Python3, later we'll create some support for Python2
+var ANACONDA_DIST_MIRROR = "https://repo.continuum.io/miniconda/Miniconda3-{{.Version}}-Linux-x86_64.sh"
```
Comment (johnewart) [inline buildpacks/anaconda.go]: Does this work for other systems or just Linux? If others can we make the URL template take those properties?

File: buildpacks/anaconda.go [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+	. "github.com/yourbase/yb/plumbing"
+	. "github.com/yourbase/yb/types"
+)
+
+type AnacondaBuildTool struct {
+	BuildTool
+	version string
+	spec    BuildToolSpec
+}
+
```
Comment (johnewart) [inline buildpacks/anaconda.go]: We should probably name this anaconda3 if the version doesn’t follow Python

File: buildpacks/anaconda.go (hunk 1 above)
Comment (beholders-eye) [reply buildpacks/anaconda.go]: I just forgot to move the variable up, I had it right. Fixing it...

**Comment by johnewart** [comment]
👍🏻

**Pull request merged**